lib/cisco_node_utils/interface_hsrp_group.rb

```diff
@@ -36,6 +36,7 @@ def initialize(interface, group_id, ip_type, instantiate=true)
 
```

Comment:
Do you need to specify `iptype:` here?  It's already one of the default keys in `set_args_keys_default` right?

https://github.com/cisco/cisco-network-node-utils/blob/3a8e8c5d4339e5faf5ab744cc413d23f3b59f3bc/lib/cisco_node_utils/interface_hsrp_group.rb#L76